Samples/Caboodle.Samples/ViewModel/CompassViewModel.cs

```diff
@@ -0,0 +1,154 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Windows.Input;
+using Microsoft.Caboodle;
+using Xamarin.Forms;
+
+namespace Caboodle.Samples.ViewModel
+{
+    class CompassViewModel : BaseViewModel
+    {
+        public CompassViewModel()
+        {
+            StartCompass1Command = new Command(async () =>
+            {
```

Comment:
! Personal Preference Warning !
Maybe this should be private methods so as not to clog the constructors?

Reply:
#resolved